src/tribler-core/tribler_core/upgrade/config_converter.py

```diff
@@ -88,6 +91,8 @@ def _fix_state_config(config):
             logger.error("Could not parse %s file on upgrade so removing it", filename)
             os.remove(filename)
 
+    return fixed_config
```

Comment:
The docstring says the function returns None, I think it is better to fix docstring or to remove the return statement

Reply:
Fixed